source/main/ipc/getHardwareWalletChannel.ts

```diff
@@ -85,22 +99,15 @@ class EventObserver {
                 AdaConnection,
               };
               this.getHardwareWalletConnectionChannel.send(
-                {
-                  disconnected: false,
-                  deviceType: 'ledger',
-                  deviceId: null,
-                  // Available only when Cardano APP opened
-                  deviceModel: deviceModel.id,
-                  // e.g. nanoS
-                  deviceName: deviceModel.productName,
-                  // e.g. Test Name
-                  path: device.path,
-                  product: device.product,
-                },
+                walletData,
                 this.mainWindow
               );
             } catch (e) {
-              logger.info('[HW-DEBUG] CONSTRUCTOR error');
+              logger.error(
+                `[HW-DEBUG] CONSTRUCTOR error, \nwalletData: ${JSON.stringify(
```

Comment:
Our logger supports passing an object as a second property, is it worth using it instead of manually serializing data?